README.md

```diff
@@ -8,88 +8,121 @@ Official [Buildkite Test Analytics](https://buildkite.com/test-analytics) collec
-      var buildkiteReporter = new BuildkiteReporter();
+   // Send results to Test Analytics
+   reporters: [
+     "default",
+     ["buildkite-test-collector/jest/reporter", { token: INSERT_TOKEN_HERE }],
+   ];
+   ```
 
-      jasmine.getEnv().addReporter(buildkiteReporter);
-    ```
+   ### Jasmine
 
-    ### Mocha
+   [Add the buildkite reporter to jasmine](https://jasmine.github.io/setup/nodejs.html#reporters):<br>
 
-    [Install mocha-multi-reporters](https://github.com/stanleyhlng/mocha-multi-reporters) in your project:<br>
+   ```js
+   // SpecHelper.js
+   var BuildkiteReporter = require("buildkite-test-collector/jasmine/reporter");
+   var buildkiteReporter = new BuildkiteReporter();
 
-    ```
-      npm install mocha-multi-reporters --save-dev
-    ```
+   jasmine.getEnv().addReporter(buildkiteReporter);
+   ```
 
-    and configure it to run your desired reporter and the Buildkite reporter
+   If you would like to pass in the API token using a custom environment variable, you can do so using the report options.
 
-    ```js
-      // config.json
-      {
-        "reporterEnabled": "spec, buildkite-test-collector/mocha/reporter"
-      }
-    ```
+   ```js
+   // SpecHelper.js
+   var buildkiteReporter = new BuildkiteReporter(undefined, {
+     token: INSERT_TOKEN_HERE,
+   });
+   ```
 
-    Now update your test script to use the buildkite reporter via mocha-multi-reporters:
+   ### Mocha
 
-    ```js
-      // package.json
-      "scripts": {
-        "test": "mocha --reporter mocha-multi-reporters --reporter-options configFile=config.json"
-      },
-    ```
+   [Install mocha-multi-reporters](https://github.com/stanleyhlng/mocha-multi-reporters) in your project:<br>
 
-4) Run your tests locally:<br>
+   ```
+     npm install mocha-multi-reporters --save-dev
+   ```
 
-    ```js
-    env BUILDKITE_ANALYTICS_TOKEN=xyz npm test
-    ```
+   and configure it to run your desired reporter and the Buildkite reporter
 
-5) Add the `BUILDKITE_ANALYTICS_TOKEN` secret to your CI, push your changes to a branch, and open a pull request 🎉
+   ```js
+     // config.json
+     {
+       "reporterEnabled": "spec, buildkite-test-collector/mocha/reporter"
+     }
+   ```
 
-    ```bash
-    git checkout -b add-bk-test-analytics
-    git commit -am "Add Buildkite Test Analytics"
-    git push origin add-bk-test-analytics
-    ```
+   Now update your test script to use the buildkite reporter via mocha-multi-reporters:
+
+   ```js
+     // package.json
+     "scripts": {
+       "test": "mocha --reporter mocha-multi-reporters --reporter-options configFile=config.json"
+     },
+   ```
+
+   If you would like to pass in the API token using a custom environment variable, you can do so using the report options.
+
+   ```js
+     // config.json
+     {
+       "reporterEnabled": "spec, buildkite-test-collector/mocha/reporter",
+       "buildkiteTestCollectorMochaReporterReporterOptions": {
```

Comment:
Would you consider removing the second `Reporter` from the string?
```suggestion
       "buildkiteTestCollectorMochaReporterOptions": {
```

Reply:
Unfortunately [as per the docs](https://github.com/stanleyhlng/mocha-multi-reporters/tree/2c511ea7847016f5be841e3e39521d944a8f6d0c#configuring-reporters) the reporter options key must reflect the reporter name in camel case ignoring all special characters and end with `reporterOptions`